### PR TITLE
Edge Storage Accelerator (ESA) - Jumpstart Scenario

### PR DESCRIPTION
**Pull request by ldabas-msft**

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+The following Jumpstart scenario will show how to create an AKS Edge Essentials single node deployment in an Azure Windows Server VM and connect the Azure VM and AKS Edge Essentials cluster to Azure Arc. The provided ARM template is responsible for creating the Azure resources as well as executing the LogonScript (AKS Edge Essentials cluster creation, Azure Arc onboarding (Azure VM and AKS Edge Essentials cluster) and Edge Storage Accelerator deployment) on the Azure VM. Once Edge Essentials is deployed [Edge Storage Accelerator](https://learn.microsoft.com/en-us/azure/azure-arc/edge-storage-accelerator/overview) is installed as a Kubernetes service that exposes a CSI driven storage class for use by applications in the Edge Essentials Kubernetes cluster.
+
+Edge Storage Accelerator (ESA) is a storage system designed for Arc-connected Kubernetes clusters to provide reliable, fault tolerant storage in a ReadWriteMany persistent volume. The Edge Storage Accelerator provides a constantly connected conduit for edge data to be replicated in cloud on blob while maintaining a local copy, as space permits, for low latency local access.
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:13]:
```suggestion
Edge Storage Accelerator (ESA) is a storage system designed for Arc-enabled Kubernetes clusters to provide reliable, fault tolerant storage in a ReadWriteMany persistent volume. The Edge Storage Accelerator provides a constantly connected conduit for edge data to be replicated in cloud on blob while maintaining a local copy, as space permits, for low latency local access. 
```

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    ```
+
+    > **Note:** If you create multiple subsequent role assignments on the same service principal, your client secret (password) will be destroyed and recreated each time. Therefore, make sure you grab the correct password.
+
+    > **Note:** The Jumpstart scenarios are designed with ease of use in-mind and adhere to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://learn.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest) as well as considering use of a [less privileged service principal account](https://learn.microsoft.com/azure/role-based-access-control/best-practices).
+
+- Create a blob storage account
+    Edge Storage Accelerator leverages a blob storage account as the final destination of data from the edge. You must create a blob storage account with or without Hierarcial Name Space (HNS) support in order to use ESA.
+
+    ```shell
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:69]: Why don't we create the storage account in the automation ?

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+    az storage container create --name esaStorageContainer
+    ```
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+- User edits the ARM template parameters file (1-time edit). These parameter values are used throughout the deployment.
+
+- Main [_azuredeploy_ ARM template](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/azuredeploy.json) will initiate the deployment of the following resources:
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:80]: Is it expected that the link points to the Akri scenario code ?

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+
+  - _Virtual Network_ - Virtual Network for Azure Windows Server VM.
+  - _Network Interface_ - Network Interface for Azure Windows Server VM.
+  - _Network Security Group_ - Network Security Group to allow RDP in Azure Windows Server VM.
+  - _Virtual Machine_ - Azure Windows Server VM.
+  - _Custom script and Azure Desired State Configuration extensions_ - Configure the Azure Windows Server VM to host AKS Edge Essentials.
+
+- User remotes into client Windows VM, which automatically kicks off the [_LogonScript_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/aks_hybrid/aks_edge_essentials_single_akri/arm_template/artifacts/LogonScript.ps1) PowerShell script to:
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:88]: Is it expected that the link points to the Akri scenario code ?

**Review by sebassem — CHANGES_REQUESTED** [review]
LGTM, some minor changes

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md (hunk 2 above)
Comment (ldabas-msft) [reply docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:69]: We do! This section is outdated, I will change it now.

**Review by dkirby-ms — CHANGES_REQUESTED** [review]

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+type: docs
+title: "Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment"
+linkTitle: "Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment"
+weight: 4
+description: >
+---
+
+## Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment 
+
+The following Jumpstart scenario will show how to create an AKS Edge Essentials single node deployment in an Azure Windows Server VM and connect the Azure VM and AKS Edge Essentials cluster to Azure Arc. The provided ARM template is responsible for creating the Azure resources as well as executing the LogonScript (AKS Edge Essentials cluster creation, Azure Arc onboarding (Azure VM and AKS Edge Essentials cluster) and Edge Storage Accelerator deployment) on the Azure VM. Once Edge Essentials is deployed [Edge Storage Accelerator](https://learn.microsoft.com/en-us/azure/azure-arc/edge-storage-accelerator/overview) is installed as a Kubernetes service that exposes a CSI driven storage class for use by applications in the Edge Essentials Kubernetes cluster.
```
[the remark this answers is not in the document]
Comment (likamrat) [reply docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:11]: - remove en-us from all links
- Make sure you have a call out that ESA is in preview. Consult with the team for examples

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+
+Azure VMs leverage the [Azure Instance Metadata Service (IMDS)](https://learn.microsoft.com/azure/virtual-machines/windows/instance-metadata-service) by default to provide information about the VMs, and to manage and configure them. By projecting an Azure VM as an Azure Arc-enabled server, a "conflict" is created which will not allow the Azure Arc server resources to be represented as one resource when the IMDS is being used. Instead, the Azure Arc server will still "act" as a native Azure VM.
+
+However, **for demo purposes only**, the below guide will allow you to use and onboard Azure VMs to Azure Arc. By doing so, you will be able to simulate a server that is deployed outside of Azure (i.e. "on-premises" or in other cloud platforms)
+
+> **Note:** It is not expected for an Azure VM to be projected as an Azure Arc-enabled server. The below scenario is unsupported and should ONLY be used for demo and testing purposes.
+
+![Screenshot ARM template output](./01.png)
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:21]: this is not the scenario architecture diagram. we created a dedicated scenario diagram as well ESA architecture diagram that you can consider including. /cc @dkirby-ms

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md (hunk 3 above)
Comment (ldabas-msft) [reply docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md]: fixed

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md (hunk 4 above)
Comment (ldabas-msft) [reply docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md]: updated

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/24.png
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/24.png:1]: old logo

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/27.png
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/27.png:1]: this looks like an Akri screenshot. Remove if not needed

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/21.png
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/21.png:1]: old logo

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/26.png
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/26.png:1]: Akri related screenshot

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/29.png
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/29.png:1]: Akri related screenshot

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/38.png
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/38.png:1]: logo is a little off center

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 7]
```diff
@@ -0,0 +1,258 @@
+---
+type: docs
+title: "Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment"
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:3]:
```suggestion
title: "Edge Storage Accelerator on AKS Edge Essentials single node deployment"
```

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+linkTitle: "Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment"
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:4]:
```suggestion
linkTitle: "Edge Storage Accelerator on AKS Edge Essentials single node deployment"
```

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 9]
```diff
@@ -0,0 +1,258 @@
...
+weight: 4
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:5]:
```suggestion
weight: 8
```

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 10]
```diff
@@ -0,0 +1,258 @@
...
+description: >
+---
+
+## Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:9]:
```suggestion
## Edge Storage Accelerator on AKS Edge Essentials single node deployment 
```

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 11]
```diff
@@ -0,0 +1,258 @@
...
+type: docs
+title: "Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment"
+linkTitle: "Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment"
+weight: 4
+description: >
+---
+
+## Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment 
+
+The following Jumpstart scenario showcases Edge Storage Accelerator (ESA), which is a storage system designed for Arc-enabled Kubernetes clusters to provide reliable, fault tolerant storage in a ReadWriteMany persistent volume. The Edge Storage Accelerator provides a constantly connected conduit for edge data to be replicated in cloud on blob while maintaining a local copy, as space permits, for low latency local access. In this scenario, a model detects defects in bolts by analyzing video from a supply line streamed over RTSP. The identified defects are then stored in a container within a storage account using ESA.
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:11]:
```suggestion
The following Jumpstart scenario showcases Edge Storage Accelerator (ESA), which is a storage system designed for Arc-enabled Kubernetes clusters to provide reliable, fault tolerant storage in a ReadWriteMany persistent volume. The Edge Storage Accelerator provides a constantly connected conduit for edge data to be replicated to blob storage in the cloud while maintaining a local copy, as space permits, for low-latency local access. 

In this scenario, a computer vision AI model detects defects in bolts by analyzing video from a supply line video feed streamed over RTSP. The identified defects are then stored in a container within a storage account using ESA.
```

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 12]
```diff
@@ -0,0 +1,258 @@
...
+
+This scenario will help you create deploy Edge Storage Accelerator by creating an AKS Edge Essentials single node deployment in an Azure Windows Server VM. The provided ARM template is responsible for creating the Azure resources as well as executing the LogonScript (AKS Edge Essentials cluster creation, Azure Arc onboarding (Azure VM and AKS Edge Essentials cluster) and Edge Storage Accelerator deployment) on the Azure VM. Once Edge Essentials is deployed [Edge Storage Accelerator](https://learn.microsoft.com/azure/azure-arc/edge-storage-accelerator/overview) is installed as a Kubernetes service that exposes a CSI driven storage class for use by applications in the Edge Essentials Kubernetes cluster.
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:13]:
```suggestion
The automation in this scenario deploys Edge Storage Accelerator on an AKS Edge Essentials single-node deployment running in an Azure virtual machine. The provided ARM template creates the Azure resources and configures the LogonScript.ps1 custom script extension which handles AKS Edge Essentials cluster creation, Azure Arc onboarding for the Azure VM and AKS Edge Essentials cluster, and Edge Storage Accelerator deployment. Once AKS Edge Essentials is deployed [Edge Storage Accelerator](https://learn.microsoft.com/azure/azure-arc/edge-storage-accelerator/overview) is installed as a Kubernetes service that exposes a CSI driven storage class for use by applications in the Edge Essentials Kubernetes cluster.
```

File: docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md [hunk 13]
```diff
@@ -0,0 +1,258 @@
...
+
+![Screenshot Azure Arc-enabled server on resource group](./22.png)
+
+- You can also run _kubectl get nodes -o wide_ to check the cluster node status and _kubectl get pod -A_ to see that the cluster is running and all the needed pods (system, [the Arc-enabled Kubernetes extension pods](https://learn.microsoft.com/azure/azure-arc/kubernetes/extensions), and [Azure Monitor extension pods](https://learn.microsoft.com/azure/azure-monitor/containers/container-insights-overview) are in a running state.
+
+![Screenshot kubectl get nodes -o wide](./36.png)
+
+![Screenshot kubectl get pod -A](./37.png)
+
+
```
Comment (dkirby-ms) [inline docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md:227]: remove extra blank line

**Review by dkirby-ms — CHANGES_REQUESTED** [review]
Looks good. Mostly minor formatting stuff. 

* Please remove all unused images including the Akri ones I flagged
* Notice I suggested change to title of the scenario
* Also changed the scenario weight to have it align in the right place in JS website.

**Review by dkirby-ms — APPROVED** [review]

**Pull request merged**